pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -148,7 +150,6 @@ type reconcileInfo struct {
 	request           reconcile.Request
 	bmcCredsSecret    *corev1.Secret
 	events            []corev1.Event
-	errorMessage      string
```

Comment:
Why is this removed?

Reply:
it is unused, so it was automatically removed by my editor. I can put it back in the next iteration

Reply:
If it's not used, removing it is fine. Maybe it could be a separate patch in this PR, though, so the history is clearer.